src/platform-apps/channels/direct-message-chat/index.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { IconXClose, IconMinus } from '@zero-tech/zui/icons';
+import classNames from 'classnames';
+import { setActiveDirectMessageId } from '../../../store/direct-messages';
+import { RootState } from '../../../store';
+import { connectContainer } from '../../../store/redux-container';
+import { ChannelViewContainer } from '../channel-view-container';
+
+import './styles.scss';
+import { DirectMessage } from '../../../store/direct-messages/types';
+
+export interface PublicProperties {}
+
+export interface Properties extends PublicProperties {
+  activeDirectMessageId: string;
+  setActiveDirectMessageId: (activeDirectMessageId: string) => void;
+  directMessage: DirectMessage;
+}
+
+interface State {
+  isFullScreen: boolean;
+  isMinimized: boolean;
+}
+
+export class Container extends React.Component<Properties, State> {
+  state = { isFullScreen: false, isMinimized: false };
+
+  static mapState(state: RootState): Partial<Properties> {
+    const {
+      directMessages: { activeDirectMessageId, list },
+    } = state;
+
+    return {
+      activeDirectMessageId,
+      directMessage: list.find((directMessage) => directMessage.id === activeDirectMessageId),
+    };
+  }
+
+  static mapActions(): Partial<Properties> {
+    return { setActiveDirectMessageId };
+  }
+
+  handleClose = (): void => {
+    this.props.setActiveDirectMessageId(null);
```

Comment:
Why are we sending null in `setActiveDirectMessageId` ?  I think the specified type is **string**.                                       
 `setActiveDirectMessageId: (activeDirectMessageId: string) => void;`